src/_gettsim/transfers/rente.py

```diff
@@ -695,6 +783,59 @@ def ges_rente_vorauss_frauen(  # noqa: PLR0913
         and ges_rente_wartezeit_15 >= 15
```

Comment:
The 15 and 10 should be parameters rather than being hard-coded here.

Apologies that this slipped through earlier, but `ges_rente_wartezeit_15` is not a great name -- `ges_rente_wartezeit_frauen_teilhabe` ? 

Probably too long for now, so we can leave it and make a note for renaming in the future. With two entries, it should be worth its own issue now :wink:

Reply:
it is actually an offical term "Wartezeit von 15 Jahren" which defines which time periods are accounted for. Similar as Wartezeit von 5 / 35 / 45 / 20 Jahren which are partly used for several pension types.

Reply:
does it help do add "_jahre"?

Reply:
My point is more that the variable name should be about what counts towards this. For example, 5 and 15 are exactly the same function right now. 

Rather than the numbers, I think something referring to the concepts might be clearer. E.g.

```py
ges_rente_wartezeit_regel >= ges_rente_params["some_threshold"]
```

`ges_rente_wartezeit_langj_versichert`
`ges_rente_wartezeit_besonders_langj_versicherte`

Probably again the length issue, so keep on piling up on that issue...

Reply:
I understand your point, but I preferred to use the same concepts that the Rentenversicherung is using and are referred to in the SGB VI. Because it is easy for the interested user to see where it comes from and what it refers to. If we make up something it will be much harder to follow.

* If we do it as you propose using functions that refer to the pension where they are used "ges_rente_wartezeit_langj_versicherte", then we would need additional functions with the same content Wartezeit_5 and wartezeit_15 are used in multiple pensions. 

* If in contrast we want to minimize the number of functions because as you say 5 and 15 are the same functions, then I would not know how to name them to keep it understandable.

One solution I can think of to avoid wartezeit_5 and 15 being the same is to change them to bool outputs by integrating the thresholds -- maybe thats what you meant with your code sample?

Reply:
Much nicer and clearer!!! Thanks!